### ISSUE TITLE
analytics

### ISSUE THREAD

**Issue by AudreyTheIndustrious** [TheIndustrious/ATT#63]
For 3PL Analytics, we need to measure the following please:
•	We need to be able to pull analytics by retailer, store location and “user.”  Remember that we are not verifying user.  But we will take all of the user values to create reporting records so that we can say that Jane uses the system 20 times per day and she sells double of Frank who uses the system once per day.
•	Broken down by store (and user in each store) or by retailer or by user
o	Sessions
o	Chapters used
o	Time/length of each session

The bottom line is that we need to understand:
1.	What content is used
2.	What retailers are being successful with it
3.	Which human beings at those retailers are using the system

**Comment by AudreyTheIndustrious** [comment]
capturing andy's comments in Git

**Comment by AudreyTheIndustrious** [comment]
Analytics is done.
